pages/docs/overview/faq.md

```diff
@@ -98,45 +73,45 @@ singularity run --bind /home/vanessa/Desktop:/data container.img
 
 
 ### How does Singularity handle networking?
-As of 2.4, Singularity now has the ability to support instances, each with their own Network namespace, and the starting behavior defined by the `%startscript`. Without running as an instance, Singularity does no network isolation and runs like any other application on the system. It has all of the same networking privileges as any program running as that user. This means that if you run a process in the container directed to localhost, it will show up on your host machine browser as localhost. Take a look at some of our <a href="https://github.com/vsoch/singularity-web" target="_blank">web examples</a> to get a sense for this.
+As of 2.4, Singularity can support the network namespace to a limited degree. At present, we just use it for isolation, but it will soon be more featurefull.
 
 
-### Can a Singularity container be multi-threaded?
+### Can Singularity support daemon processes?
+Singularity has container "instance" support which allows one to start a container process, within its own namespaces, and use that instance like it was a stand alone, isolated system.
+
+At the moment (as above describes), the network (and UTS) namespace is not well supported, so if you spin up a process daemon, it will exist on your host's network. This means you can run a web server, or any other daemon, from within a container like and access it directly from your host.
```

Comment:
(as described above)

Reply:
Ooops, is this redundant?

Reply:
no I think it's ok, I wanted to suggest changing (as above described) to (as described above) because it's more commonly used